roles/ara/tasks/main.yml

```diff
@@ -0,0 +1,42 @@
+---
+ ### Update apt cache, but don't mark changed
+ # Works around Ansible 2.2 bug
+ # TODO: Remove with Ansible 2.3
+- name: Update apt cache
+  apt:
+    update_cache: yes
+  changed_when: False
+
+- name: Install deps
+  apt:
+    name: "{{ item }}"
+    state: installed
+  with_items:
+    - gcc 
+    - libffi-dev 
+    - python-dev
+    - libssl-dev
+
+- name: Install ara
+  pip:
+    name: ara
+
+- name: Ensure ara directory
+  file:
+    path: /var/www/ara/
+    state: directory
+    mode: 0755
+    owner: www-data
+    group: www-data
+
+- name: Copy ARA wsgi script into place
+  command: cp -p /usr/local/bin/ara-wsgi /var/www/ara/
```

Comment:
Yeah, I took a few shortcuts that I know I need to go back to and clean up. I'll do that shortly.